channels/microsoft365/m365-graph/index.rst

```diff
@@ -0,0 +1,129 @@
+Microsoft 365 Graph Email
+=========================
+
+You can connect to user and shared mailboxes in your M365 environment.
+Follow the steps below and skip not applicable parts.
+
+First Steps
+-----------
+
+- Check your FQDN under *Settings > System > Fully Qualified Domain Name* in the
+  admin interface of Zammad. If it is not correct, change it now. Otherwise the
+  setup of the channel will fail.
+- Go to *Channels > Microsoft 365 Graph Email* and click on
+  **Connect Microsoft 365 App**. Copy the provided callback URL.
+
+Configuration
+----------------
+
+Go to Microsoft's `Entra admin center <https://entra.microsoft.com/#home>`_ and
+log in with as an administrator.
+
+Create App
+^^^^^^^^^^
+
+- Create a new app by going to *Applications > App registrations* and select
+  **New registration**.
+- Enter a fitting name and select an account type. Supported types are:
+
+  - Accounts in this organizational directory only (Single tenant)
+  - Accounts in any organizational directory (Multitenant)
+
+- Under "Redirect URI", select "Web" as platform and paste your already copied
+  callback URL from Zammad.
+- Click on **Register**.
+- In the overview screen, copy the "Application (client) ID", switch to Zammad
+  and paste it in the "Client ID" field in the pop up.
+- Only required for single tenant setup: copy the "Directory (tenant) ID" and
+  paste it in the "Tenant UUID/Name" field in Zammad.
+
+Create Secret
+^^^^^^^^^^^^^
+
+- In Entra, go to "Certificates & secrets" and add a secret by clicking the
+  **New client secret** button.
+- Enter a description, set an expiry duration and click **Add**.
+- Copy the string under "Value", this is the secret. Paste it to Zammad
+  in the "Client Secret" field.
+
+Configure API Permissions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Go to "API permissions" and **Add a permission**.
+- Select "Microsoft Graph" and "Delegated permissions".
+- Add the following permissions:
+
+  - ``offline_access``
+  - ``openid``
+  - ``profile``
+- Add the following permissions for **user mailboxes** only:
+
+  - ``Mail.ReadWrite``
+  - ``Mail.Send``
+- Add the following permissions for **shared mailboxes** only:
+
```

Comment:
Please drop these lines for now. We decided to base the scope on a static list of permissions, so all would be needed regardless of the mailbox type.